src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { Todo } from '../../types/todo';
+
+interface Props {
+  todos: Todo[];
+  onDelete: CallableFunction;
```

Comment:
👀 🔥 
I has never known about such way of type assigning. Looks great and clear.